pkg/client/unversioned/record/event_test.go

```diff
@@ -316,17 +319,17 @@ func validateEvent(actualEvent *api.Event, expectedEvent *api.Event, t *testing.
 	if actualEvent.FirstTimestamp.IsZero() || actualEvent.LastTimestamp.IsZero() {
 		t.Errorf("timestamp wasn't set: %#v", *actualEvent)
 	}
-	if actualEvent.FirstTimestamp.Equal(actualEvent.LastTimestamp) {
+	actualFirstTimestamp := actualEvent.FirstTimestamp
```

Comment:
This isn't a go1.5 workaround/fix. Just a small code simplification I did along the way while I fixed the race bug.